library/src/test/java/com/datatorrent/lib/helper/OperatorContextTestHelper.java

```diff
@@ -44,22 +42,26 @@ protected DateFormat initialValue()
 
   };
 
-  public static class TestIdOperatorContext extends TestContext implements OperatorContext
+  public abstract static class TestIdOperatorContext extends TestContext implements OperatorContext
```

Comment:
I disagree. Test classes should use OperatorContext interface only and exposing TestIdOperatorContext promote incorrect usage. There is no need for extra import statement. The method can be imported statically.